security/encrypt-cluster-pod-traffic.md

```diff
@@ -187,11 +161,26 @@ calicoctl patch felixconfiguration node.<Node-Name> --type='merge' -p '{"spec":{
 To disable encryption for pod traffic on node `my-node`, use the following command:
 
   ```bash
-calicoctl patch felixconfiguration node.my-node --type='merge' -p '{"spec":{"wireguardEnabled":false}}'
+cat <<EOF | kubectl apply -f -
+apiVersion: projectcalico.org/v3
+kind: FelixConfiguration
+metadata:
+  name: node.my-node
+spec:
+  logSeverityScreen: Info
+  reportingInterval: 0s
+  wireguardEnabled: true
```

Comment:
Isn't this supposed to be false ? I think we want to disable it.

Reply:
Also, why are we not using a patch command for the felix config ?

Reply:
ah good catch, the gist is the per-node config has to be existing before it can be patched